libraries/abstractions/pkcs11/test/MBT_C_GenerateRandom.c

```diff
@@ -38,7 +38,7 @@ void C_GenerateRandom_normal_behavior()
 {
     CK_SESSION_HANDLE hSession = xGlobalSession;
     CK_BYTE pRandomData_val[100] = { 0 };
-    CK_BYTE_PTR pRandomData = &pRandomData_val;
+    CK_BYTE_PTR pRandomData = pRandomData_val;
```

Comment:
Similar to above, could replace the redundant hard-coded length literal with a sizeof call.

Reply:
Changed in 8ccfd49.